src/compiler/types.ts

```diff
@@ -5844,7 +5844,8 @@ namespace ts {
         enable?: boolean;
         include?: string[];
         exclude?: string[];
-        [option: string]: string[] | boolean | undefined;
+        inferTypingsFromFilenames?: boolean;

```

Comment:
I think you want apposite of this option so that current type acquisitions in config file behave as if this option is set to false. Ideally you want new behavior to kick in only when the option is set

Reply:
I _think_ I accounted for that, making so that the behavior is only disabled if explicitly set to `false`. I was imagining something like `Enable/disable filename-based ATA` as the text of the setting on the editor side, with it checked by default. 

I suppose `disableFilenameBasedTypeAcquisition` works just as well.